misc/python/materialize/feature_benchmark/benchmark.py

```diff
@@ -155,23 +156,41 @@ def run(self) -> list[Aggregation]:
                 print(f"{i}: {messages_measurement}")
                 self._messages_aggregation.append(messages_measurement)
 
-            if self._memory_aggregation:
-                memory_measurement = Measurement(
-                    type=MeasurementType.MEMORY,
-                    value=self._executor.DockerMem() / 2**20,  # Convert to Mb
+            if self._memory_mz_aggregation:
+                memory_mz_measurement = Measurement(
+                    type=MeasurementType.MEMORY_MZ,
+                    value=self._executor.DockerMemMz() / 2**20,  # Convert to Mb
                 )
 
-                if memory_measurement.value > 0:
-                    if not self._filter or not self._filter.filter(memory_measurement):
-                        print(f"{i} {memory_measurement}")
-                        self._memory_aggregation.append(memory_measurement)
+                if memory_mz_measurement.value > 0:
+                    if not self._filter or not self._filter.filter(
+                        memory_mz_measurement
+                    ):
+                        print(f"{i} {memory_mz_measurement}")
+                        self._memory_mz_aggregation.append(memory_mz_measurement)
+
+            if self._memory_clusterd_aggregation:
+                memory_clusterd_measurement = Measurement(
+                    type=MeasurementType.MEMORY_CLUSTERD,
+                    value=self._executor.DockerMemClusterd() / 2**20,  # Convert to Mb
+                )
+
+                if memory_clusterd_measurement.value > 0:
+                    if not self._filter or not self._filter.filter(
+                        memory_clusterd_measurement
+                    ):
+                        print(f"{i} {memory_clusterd_measurement}")
+                        self._memory_clusterd_aggregation.append(
+                            memory_clusterd_measurement
+                        )
```

Comment:
Optional: This snippet is rather similar to the previous one. Can we extract a function?

Reply:
I thought about that too, but probably increases the complexity more than it helps.